### DISCUSSION TITLE
Dashboard balance does not match wallet balance.

### DISCUSSION THREAD

**Discussion by SireWilliamson** [btcpayserver/btcpayserver#5304] [Technical Support]
I recently migrated from DBTrie for NBXPLORER to postgres, now the balance of the wallet on the dashboard is significantly lower that the actual wallet balance.
When I click on the bitcoin wallet for the store, it shows the correct balance.
How do I fix this?

**Comment by dennisreimann** [comment]
Ping @NicolasDorier

**Comment by NicolasDorier** [comment]
this is weird, I believe both are using the same underlying code. May it be a cache issue?
@SireWilliamson did you tried to refresh the page, is it still an issue?

**Reply by SireWilliamson** [reply]
Yes. I cleared my cache. I cleared the server cache. Does the dashboard give the whole history, or pull the balance from only the last years transactions?

**Comment by Johnjrlee** [comment]
I have tried everything if there's any way y'all can maybe give me a call
and help me and walk me through it my number is 904-577-1743 because I
cannot find the number to get in touch with y'all
